### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by Jeb-DELETE-THIS-ACCOUNT-EVENTUALLY**
With this branch merged into master, all of classes.json should be properly spellchecked now.  Please @ me over Discord and tell me if I missed anything so I can kick myself for it a month later.

### CODE REVIEW

**Comment by Buluphont** [comment]
Excerpt from Discord discussion regarding this issue, posted here by Jeb's request. Agrees with @TheGiddyLimit 's concerns and examples.

```
[11:38 AM] Marisa, tbh: Oh, I think I see the rule
[11:39 AM] Marisa, tbh: So when you use Nth-level to describe something immediately after
[11:39 AM] Marisa, tbh: You use a hyphen
[11:39 AM] Marisa, tbh: In other words, as an adjective immediately preceding its descriptee
[11:39 AM] Marisa, tbh: So, always Nth-level thing
[11:39 AM] Marisa, tbh: But when you're talking about the level itself
[11:39 AM] Marisa, tbh: In other words
[11:39 AM] Marisa, tbh: Your spells of Nth level and higher
[11:39 AM] Marisa, tbh: You don't use the hyphen
[11:39 AM] Marisa, tbh: That's what I can divine from this single excerpt
[11:40 AM] Marisa, tbh: If you have good examples of other excerpts to analyze, I can refine
[11:40 AM] Marisa, tbh: But that seems to me what WotC writers are trying to do
```

**Comment by Jeb-DELETE-THIS-ACCOUNT-EVENTUALLY** [comment]
Thank you both so much for your elaboration.  It seemed inconsistent when I was going through it all.  I was confused by these hyphens and unsure of when the _xth level_ phrase was to be hyphenated.  It seemed inconsistent, but now I see that it's about what comes after the word level.  I will commit these fixes and then it should finally be ready to merge!  Well, after examining the remaining commits of course.  I'm rambling, sorry.

File: data/classes.json [hunk 1]
```diff
@@ -7482,10 +7484,10 @@
...
                          null,
-                         "Tenets of Conquest",
+                         "Tenets of Conquest:",
                          null,
                          "A paladin who takes this oath has the tenets of conquest seared on the upper arm. A hell knight's oath appears in Infernal runes, a brutal reminder of vows to the Lords of Hell.",
-                         "Douse the Falem of Hope. It is not enough to merely defeat an enemy in battle. Your victory must be so overwhelming that your enemies will to fight is shattered forever. A blade can end a life. Fear can end an empire",
```
Comment (Jeb-DELETE-THIS-ACCOUNT-EVENTUALLY) [inline data/classes.json]: The funniest error I've seen while going through this: here on line 7488.

**Comment by Jeb-DELETE-THIS-ACCOUNT-EVENTUALLY** [comment]
It is done!  Please merge this branch into master.

File: data/classes.json [hunk 2]
```diff
@@ -15115,7 +15126,7 @@
                            "01-15 Potion of greater healing",
                            "16-22 Potion of fire breath",
                            "23-29 Potion of resistance",
-                           "30-34 Ammunition , +1",
+                           "30-34 Ammunition +1",
```
Comment (Buluphont) [inline data/classes.json]: Is this intentional? I thought that `{Item Name}, +{N}` was the standard for enhanced magical items in tables like this. Do correct me if I'm mistaken.

IE `Longsword, +2`

**Comment by astranauta** [comment]
👌

**Pull request merged**